plugins/harness-terraform/content.yaml

```diff
@@ -1,17 +1,16 @@
-title: terraform-plan
-hide: true
-author: drone-plugins
+title: Harness Terraform
+author: Harness
 tags:
   - infrastructure
   - harness
   - terraform
 logo: terraform.svg
 repo: https://github.com/drone-plugins/drone-terraform
-image: https://hub.docker.com/r/plugins/drone-terraform
+image: https://hub.docker.com/r/plugins/harness_terraform:linux-amd64
 license: None
-readme: https://github.com/drone-plugins/drone-terraform
+readme: https://github.com/drone-plugins/drone-terraform/tree/main#readme
```

Comment:
can we change this to just `image: https://github.com/drone-plugins/drone-terraform/blob/main/README.md`